aiidalab_widgets_base/computational_resources.py

```diff
@@ -852,19 +852,26 @@ def _configure_computer(self, computer: orm.Computer):
 
         return True
 
+    def _run_things_on_existing_computer(self, label):
```

Comment:
Could the name be a more descriptive. It is not clear at all what are the "things". 
What are the examples of functions that run on `_on_computer_success`?

Reply:
Maybe `_run_callbacks_on_existing_computer()`?

Reply:
Hmm, but then should these registered function even run if the computer already exist?
Supposedly refresh would not be needed in this case...

I guess the name should be `_run_callbacks_if_computer_exists`, otherwise it implies we're are running something on the remote computers...